network/manager.go

```diff
@@ -115,7 +115,7 @@ type NetworkManager interface {
 	DetachEndpoint(networkID string, endpointID string) error
 	UpdateEndpoint(networkID string, existingEpInfo *EndpointInfo, targetEpInfo *EndpointInfo) error
 	GetNumberOfEndpoints(ifName string, networkID string) int
-	GetEndpointID(containerID, ifName string) string
+	GetEndpointID(containerID, ifName string, nicType cns.NICType) string
```

Comment:
Do we have any tests?

Reply:
Unfortunately we don't. I am planning to add a subsequent (sister) PR to this that add UTs for some of these funcs.